app/v5/createForeignPropertyPeriodCumulativeSummary/def1/model/request/PropertyIncome.scala

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v5.createForeignPropertyPeriodCumulativeSummary.def1.model.request
+
+import play.api.libs.functional.syntax._
+import play.api.libs.json.{JsPath, Json, Reads, Writes}
+
+case class PropertyIncome(rentIncome: Option[RentIncome],
+                          foreignTaxCreditRelief: Option[Boolean],
+                          premiumsOfLeaseGrant: Option[BigDecimal],
+                          otherPropertyIncome: Option[BigDecimal],
+                          foreignTaxPaidOrDeducted: Option[BigDecimal],
+                          specialWithholdingTaxOrUkTaxPaid: Option[BigDecimal])
+
+object PropertyIncome {
+  implicit val reads: Reads[PropertyIncome] = Json.reads[PropertyIncome]
+
+  implicit val writes: Writes[PropertyIncome] = (
+    (JsPath \ "rentIncome").writeNullable[RentIncome] and
+      (JsPath \ "foreignTaxCreditRelief").writeNullable[Boolean] and
+      (JsPath \ "premiumsOfLeaseGrant").writeNullable[BigDecimal] and
```

Comment:
premiumsOfLeaseGrant and otherPropertyIncome have different names downstream and will have to be mapped like the expenses fields.